### PR TITLE
[bcq-tools] Add missing license in bcq-tools

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit will add missing license in `bcq-tools`

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: compiler/bcq-tools/generate_bcq_output_arrays [hunk 1]
```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python3
 
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+# Copyright (C) 2018 The TensorFlow Authors
```
Comment (seanshpark) [inline compiler/bcq-tools/generate_bcq_output_arrays:4]: does this code has direct copy from tensorflow?

Comment (llFreetimell) [reply compiler/bcq-tools/generate_bcq_output_arrays:4]: There is no direct code from Tensorflow but the codes import TensorFlow. So I thought that we should write the license author like `tf2tfliteV2.py`

Comment (seanshpark) [reply compiler/bcq-tools/generate_bcq_output_arrays:4]: tf2tfliteV2.py has copied code from TensorFlow :)
If there is no direct copy code then we can skip this if you agree.

Comment (llFreetimell) [reply compiler/bcq-tools/generate_bcq_output_arrays:4]: Oh, I didn't know that @.@
Frankly speaking, I totally agree with you but I want to be checked that we are really correct.
So after it is checked, I will remove it :)

**Comment by llFreetimell** [comment]
This will be handled in another PR. Close this!

**Pull request closed without merging**